{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -51,3 +52,13 @@ docs: ## Build docs using Sphinx and copy to docs folder (this makes it easy to
 
 ipython: ## Provides an interactive ipython prompt
 	docker exec -it $(CONTAINER_NAME) ipython
+
+setup-dvc: ## Initializes DVC in project repo
+	docker exec -it $(CONTAINER_NAME) dvc init
+	git add .dvc .dvcignore
+	git commit -m "Initialize DVC in repo"
+
+dvc-remote: ## Adds DVC remote to repo.
+	docker exec -it $(CONTAINER_NAME) dvc remote add --default default_remote $(DVC_REMOTE_URL)
+	git add .dvc/config
```

Comment:
I don't love adding `git` commands here (same with `setup-dvc`). If I run this again, I don't want to add another git commit.

Reply:
Yeah, agreed it's non-explicit behavior. I'll remove it.